### PR TITLE
Add Allowed Host Paths PSP

### PR DESCRIPTION
**Pull request by EmandM**

### CODE REVIEW

File: Policies/Kubernetes/allowed-host-paths/rego/src_test.rego [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+        "metadata": {
+            "name": "nginx"
+        },
+        "spec": {
+            "initContainers": input_containers_many,
+            "volumes": input_volumes_many
+      }
+    }
+}
+
```
Comment (RamyasreeChakka) [inline Policies/Kubernetes/allowed-host-paths/rego/src_test.rego:140]: Can you please add tests for below input review types...
1. input review with just containers and no volumes
1. input review with containers and volumes but volumes do not have host path in it.

File: Policies/Kubernetes/allowed-host-paths/rego/src_test.rego [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+test_input_hostpath_allowed_writable_not_allowed {
+    input := { "review": input_review_writable, "parameters": input_parameters_in_list}
+    results := violation with input as input
+    count(results) == 1
+}
+test_input_hostpath_not_allowed_writable {
+    input := { "review": input_review_writable, "parameters": input_parameters_not_in_list}
+    results := violation with input as input
+    count(results) == 1
+}
```
Comment (RamyasreeChakka) [inline Policies/Kubernetes/allowed-host-paths/rego/src_test.rego:32]: Can you add tests with input_review_writable and input_parameters_in_list_writable?

File: Policies/Kubernetes/allowed-host-paths/azurepolicy.json [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      },
+      "excludedNamespaces": {
+        "type": "Array",
+        "metadata": {
+          "displayName": "Namespace exclusions",
+          "description": "List of Kubernetes namespaces to exclude from policy evaluation. Providing a value for this parameter is optional."
+        },
+        "defaultValue": []
+      },
+      "allowedHostPaths": {
```
[the remark this answers is not in the document]
Comment (EmandM) [reply Policies/Kubernetes/allowed-host-paths/azurepolicy.json:33]: It allows an empty list fine

[the remark this answers is not in the document]
Comment (RamyasreeChakka) [reply Policies/Kubernetes/allowed-host-paths/azurepolicy.json:33]: Awesome!

**Pull request merged**